src/org/sosy_lab/cpachecker/cfa/parser/llvm/CFABuilder.java

```diff
@@ -1748,11 +1748,7 @@ private CExpression createGetElementPtrExp(final Value pItem, final Path pFileNa
 
     if (pItem.canBeTransformedFromGetElementPtrToString()) {
       String constant = pItem.getGetElementPtrAsString();
-      CType constCharType =
-          new CSimpleType(
-              true, false, CBasicType.CHAR, false, false, false, false, false, false, false);
-
-      CType stringType = new CPointerType(false, false, constCharType);
+      CType stringType = new CPointerType(false, false, CNumericTypes.CONST_CHAR);
```

Comment:
Please check whether it is applicable for eclipse parser with pointers to strings

Reply:
Replaced new CPointerType instance with a static one. (Still do not understand the question)